### DISCUSSION TITLE
exclude_patterns

### DISCUSSION THREAD

**Discussion by chipbrock** [executablebooks/meta#393] [Q&A]
I'm moving a large Bookdown project into Jupyter books and so I'm looking for like-minded tools. One is the ability to work on one file at a time in a multi-file textbook. My understanding is that to do this in jupyter-books is to add to the _config.yml:

`execute:`
  `execute_notebooks: auto`
  `exclude_patterns: [randomly/*]`

because I have markdown files inside of the directory `randomly' and I would like to ignore execution of all of them. This doesn't work...all are executed anyway.

If I do 

`execute:`
  `execute_notebooks: auto`
  `exclude_patterns: [randomly/content]`

as a test to ignore one of the files called `content` it still executes and renders that file also. 

clearly I'm not getting something here and I'd appreciate advice. (in bookdown... it's command-K on a mac to just process the open file).

Thanks.

**Comment by choldgraf** [comment]
hmm - could you go into the use-case you're shooting for a bit more? You *should* be able to only edit a single file, re-build the book, and only that file will re-build. But it sounds like all of them are rebuilding for you regardless?

**Reply by chipbrock** [reply]
Yes, that's exactly the case. Those lines in the _config yaml file I believe worked for me before, but then I went through the confusion with Jupyter book upgrade and the _toc problem. Now it's ignored and all files are executed in the /randomly/ directory. This is a Sphinx thing, right? Could I have some version mixup there?

Thanks for your help.
